Resources/config/services.xml

```diff
@@ -115,7 +115,7 @@
 
         <!-- List Builder -->
         <service id="sulu_form.list_builder.dynamic_list_factory"
-                 class="Sulu\Bundle\FormBundle\ListBuilder\DynamicListFactory">
+                 class="Sulu\Bundle\FormBundle\ListBuilder\DynamicListFactory" public="true">
```

Comment:
Do you still need these services to be public? Please check all services in this file.

Reply:
- `DynamicListMetadataLoader` and `DynamicFormMetadataLoader`: removed public and defined services for tests in `config.yml`
- `DynamicController`: needs to be public
- `DynamicRepository`: removed public
- `DynamicListFactory`: removed public
- `FormManager`: needs to be public